index.html

```diff
@@ -3096,6 +3096,15 @@ <h2 id="faq">F.A.Q.</h2>
       replacing with the path to your config file, and the config file is valid, the test suite should run properly.
     </p>
 
+      <b class="header">My tests are failing because slow DB connection and short test timeouts! How to extend test timeouts?</b><br />
+      Some times e.g. with running CI on travis test suites default timeout 5seconds might be too short. The test suite looks from an environment variable 
+      called <tt>KNEX_TEST_TIMEOUT</tt> for milliseconds how long each test might run before timing out.
```

Comment:
Suggested doc text update:

> Sometimes, e.g. when running CI on travis, test suite's default timeout of 5 seconds might be too short. In such cases an alternative test timeout value in milliseconds can be specified using the <tt>KNEX_TEST_TIMEOUT</tt> environment variable.

(hmmm, I already wrote up this comment before but it's not here now... must not have confirmed it or something...)